.gitignore

```diff
@@ -133,3 +133,6 @@ dask-worker-space/
 
 # Generic data directories
 **/data/*
+
+# Modeling
+checkpoint/
```

Comment:
the default is `checkpoints` https://github.com/HazyResearch/snorkel/blob/redux/snorkel/classification/training/loggers/checkpointer.py#L38

Reply:
I think this is Keras maybe? Idk evidently something I did made this dir

Reply:
possibly—which tutorial were you running? 
in any case, let's either include both checkpoint dirs or just one of them and make it consistent across tutorials (cc @bhancock8, for visibility when we check consistency)